portality/models/background.py

```diff
@@ -25,6 +28,9 @@ def __init__(self, **kwargs):
 
         super(BackgroundJob, self).__init__(raw=kwargs)
 
+        if not self.outcome_status:
+            self.outcome_status = BgjobOutcomeStatus.Pending
```

Comment:
should we use 'value' attribute for Pending?

Reply:
reason same as above